tests/performance/locustfiles/deployment_max_rps_single_endpoint.py

```diff
@@ -52,4 +65,11 @@ def get_endpoint(self) -> None:
         kwargs = {}
         if len(self.environment.parsed_options.body) > 0:
             kwargs["json"] = json.loads(self.environment.parsed_options.body)
+        if len(self.environment.parsed_options.body_json_schema) > 0:
+            faker = jsf.JSF(
+                json.loads(self.environment.parsed_options.body_json_schema)
+            )
+            kwargs["json"] = faker.generate()
```

Comment:
looks too magic to me :-D

Another good use case for this library is e.g. to find out how to constraint specially input schemas to public APIs. There are some security checks we do but thisis another way to fine tune constraints